### ISSUE TITLE
Ethereum lost during failed Wallet creation (2 Times)

### ISSUE THREAD

**Issue by Wichipingui** [ethereum/mist#3053]
```
Version: `0.90`
OS & Version: windows 7 x64
Node version: `geth 1.6.6` 
Number of blocks synchronized: 4288650
```
Hi, today I tried to create a Wallet (Simple version, not multi-user) and it hanged for a while instead of creating it. So I thought something went wrong so I closed the software (Ethereum Wallet) and I tried the process once again. It didnt work again (I waited for more than 30 Minutes), and then I saw that they charged me the price of two wallets for my failed attempts. Thats aprox. 0,08 Ether. So I want to know how do I recover my Ethereum please?

Thanks in advance for your help, I dont know what else to do :(

![b7f4f94a-4438-408c-9672-716aad614819](https://user-images.githubusercontent.com/32077351/30563549-836b3da4-9cc2-11e7-8a3c-1258e4fe6a35.jpg)
[nodelogs.zip](https://github.com/ethereum/mist/files/1312159/nodelogs.zip)

**Comment by riartem** [comment]
I've seen similar bug when toggle Wallet to test network and back again. Accounts disappeared.
Seems like official client too ugly and not ready for production yet.

**Comment by evertonfraga** [comment]
That behavior was fixed in 0.9.3.
